packages/react/src/hig-react.js

```diff
@@ -56,7 +56,7 @@ export {
   default as RadioButton
 } from "./adapters/FormElements/RadioButtonAdapter";
 export { default as Range } from "./adapters/FormElements/RangeAdapter";
-export { default as RichText } from "./elements/components/RichText";
+export { default as RichText } from "./elements/components/RichText/RichText";
```

Comment:
Thoughts on renaming the component to `packages/react/src/elements/components/RichText/index.js` instead? Then the import path can remain the same.